chainerrl/wrappers/continuing_time_limit.py

```diff
@@ -46,3 +51,76 @@ def step(self, action):
     def reset(self):
         self._elapsed_steps = 0
         return self.env.reset()
+
+
+class ContinuingTimeLimitMonitor(Monitor):
+    """`Monitor` with ChainerRL's `ContinuingTimeLimit` support.
+
+    Because of the original implementation's design,
+    explicit `close()` is needed to save the last episode.
+    Do not forget to call `close()` at the last line of your script.
+
+    For details, see
+    https://github.com/openai/gym/blob/master/gym/wrappers/monitor.py
+    """
+
+    def _start(self, directory, video_callable=None, force=False, resume=False,
+               write_upon_reset=False, uid=None, mode=None):
+        if self.env_semantics_autoreset:
+            raise gym.error.Error(
+                "Detect 'semantics.autoreset=True' in `env.metadata`, "
+                "which means the env comes from deprecated OpenAI Universe.")
```

Comment:
> OpenAI Universe is deprecated, but Gym still supports `semantics.autoreset=True`, correct?

I think so.

> If so, I think we should state here that `ContinuingTimeLimitMonitor` does not support it.

OK, I'll add doc for that.

> In addition, I think we should not use `gym.error.Error` since it is us, not gym, who raises that error.

I think we can use `gym.error.Error` because `ContinuingTimeLimitMonitor` inherits  `gym.Monitor`, but it is okay to raise `ValueError` or `NotImplementedError` for this `if` clause in order to clarify that we do not support Universe.